okapi-operation/src/axum_integration/axumyaml.rs

```diff
@@ -0,0 +1,34 @@
+use axum::response::{IntoResponse, Response};
+use bytes::{BufMut, BytesMut};
+use http::{header, HeaderValue, StatusCode};
+use serde::Serialize;
+
+pub struct AxumYaml<T>(pub T);
+
+impl<T> IntoResponse for AxumYaml<T>
+where
+    T: Serialize,
+{
+    fn into_response(self) -> Response {
+        let mut buf = BytesMut::with_capacity(128).writer();
+        match serde_yaml::to_writer(&mut buf, &self.0) {
+            Ok(()) => (
+                [(
+                    header::CONTENT_TYPE,
+                    HeaderValue::from_static("text/x-yaml"),
+                )],
+                buf.into_inner().freeze(),
```

Comment:
Shouldn't this work with just `buf`? IntotResponse is [implemented](https://docs.rs/axum/latest/axum/response/trait.IntoResponse.html#impl-IntoResponse-for-BytesMut) for BytesMut.

Reply:
We also need Writer instance to write the actual yaml into the response.
And actually this is just a bit modified copy paste from Json response in Axum